src/Players.java

```diff
@@ -0,0 +1,27 @@
+public class Players {
+
+    int points = 0;
+
+    public Players() {
+
+    }
+
+    public void add () {
+        points = points +1;
+    }
+
+    public void sub () {
+        points = points - 1;
+    }
+
+    public void reset () {
+        points = 0;
+    }
+
+
+    public int getter () {
```

Comment:
Maybe we can rename this to something like `getScore` or `getPoints`, `getter` is a bit generic

Reply:
# Sure!
 it was just practically for me to __understand__ what a getter method is